components/Providers/Dispatcher.php

```diff
@@ -18,4 +18,11 @@ public function boot()
 
         $dispatcher->setEventsManager($event_manager);
     }
+    
+    /**
+     * Override action suffix default phalcon from `action` to ``
+     */
+    public function getActionSuffix(){
```

Comment:
please make the curly opening in the nextline to support PSR. I will merge it immediately once you fixed it :+1: